libraries/Microsoft.Bot.Builder.AI.QnA/Helpers/TrainHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Helper class for train API
+    /// </summary>
+    public class TrainHelper
+    {
+        private static readonly string QnAMakerName = nameof(QnAMaker);
+        private QnAMakerEndpoint _endpoint;
+        private readonly HttpClient httpClient;
```

Comment:
Same comment here - use the HTtpClientHandler, as it's faster and the "Current" way of doing things. We've got a number of examples in our codebase.